src/GslibIO.jl

```diff
@@ -7,12 +7,16 @@ module GslibIO
 using FileIO
 using Printf
 using DelimitedFiles
+using DataFrames
 
 using GeoStatsBase
 
 # type aliases
 const Array2or3{T} = Union{AbstractArray{T,2},AbstractArray{T,3}}
 
+# For naming coordinates in a grid
+const DefaultCoordNames = [:x, :y, :z]
+
```

Comment:
No existing function is using :x, :y or :z, because you are working with grid and you don't need to store coordinates.
They are needed when you store a pset for naming the variables that are coordinates. Maybe I can define it inside the function instead.

Reply:
Yes, that would be a great. Just pass these names as a default option instead of creating an external variable :+1: